flexget/utils/tools.py

```diff
@@ -160,12 +276,12 @@ def merge_dict_from_to(d1, d2):
                     merge_dict_from_to(d1[k], d2[k])
                 elif isinstance(v, list):
                     d2[k].extend(copy.deepcopy(v))
-                elif isinstance(v, (basestring, bool, int, float, type(None))):
```

Comment:
You noted this in my other plugin, so this is only inside plugins and not tools?

> `'guid': basestring,`
We should pretend that this is python 3, and using 'str' instead of basestring (our from builtins import * makes this a real python 3 str (text) even on python 2.) If there is an actual possibility that the thing will be either text (str) or bytes (bytes), we should handle those cases explicitly.